Dfe.Academies.Academisation.IService/Commands/Legacy/Project/ILegacyProjectDeleteNoteCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Dfe.Academies.Academisation.Core;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate;
+
+namespace Dfe.Academies.Academisation.IService.Commands.Legacy.Project
+{
+	public interface ILegacyProjectDeleteNoteCommand
+	{
+		Task<CommandResult> Execute(int projectId, ProjectNoteServiceModel note);
```

Comment:
Why is there two DeleteNote commands?

Reply:
This is the crux of the refactor, really. The refactor was because I had used the `AcademisationContext` outside of the data project/layer which is undesirable.

So; moved the command to the data layer (`ProjectNoteAddCommand` and `ProjectNoteDeleteCommand` where access to the `AcademisationContext` doesn't cross project boundaries) but the commands used at "higher" levels of the codebase are in the `Service` project. So - to keep the codebase consistent - there are these small wrapper commands here, too.

I suppose you could think of it like this: the ones in the `Data` project are the data commands, the ones in the service are the "domain action" commands and the controllers use domain action commands, not data commands directly.

Reply:
The logic makes sense but can't say I love the pattern!

Reply:
Granted, in this case, the code is that "thin" that this looks odd, so is definitely more for wider codebase design consistency.